packages/main/src/themes/Link.css

```diff
@@ -11,7 +11,6 @@
 	font-size: var(--sapFontSize);
 	cursor: pointer;
 	outline: none;
-	text-decoration: var(--_ui5_link_text_decoration);
```

Comment:
I wanted to comment on one aspect, that you may haven't been aware of - the text-decoration, as the colour and font-size, have been all set on the host element to allow users change them easily from outside:

```html
<ui5-link style="color: red, text-decoration: yellow"></ui5-link>
```
Moving it to the root element won't allow it anymore.

Also these selectors have been also changed from the host
```css
:host(:hover) {
	color: var(--sapLink_Hover_Color);
	text-decoration: var(--_ui5_link_hover_text_decoration);
}

:host([design="Subtle"]:hover) {
	color: var(--sapLink_SubtleColor);
	text-decoration: var(--_ui5_link_subtle_text_decoration_hover);
}
```

to apply on the .ui5-link-root, and perhaps the "colour" won't be customisable as in the aforementioned way (with standard CSS on the element tag).

Could you test the custamization of color and text-decoration
in the test page, I think these cases should be sufficient to see if the customazion is affected
```css
ui5-link {
color: red;
text-decoration: yellow
}

ui5-link:hover {
color: purle;
text-decoration: darkblue;
}

ui5-link[design="Subtle"]:hover {
color: green;
text-decoration: black
}
```

Reply:
See this selector
```css
.ui5-link-root {
	width: 100%;
	display: inline-block;
	position: relative;
	overflow: hidden;
	text-overflow: ellipsis;
	outline: none;
	white-space: inherit;
	overflow-wrap: inherit;
	text-decoration: inherit;
	color: inherit;
}
```

all these with the "inherit" value are supposed to be inherited from the host.

Reply:
Yes, it is all clear now. Did not have that in mind at all. @didip1000 will correct those selectors after she gets back in the office.

Reply:
I switched those to the root because I was having issues with the hover effect taking precedence over the active state. I can check when I get back from my vacation if there's any other way I can fix that :)

Reply:
I just pushed a new commit that should fix this, hopefully :)